ote/test/clj/ote/tasks/pre_notices_test.clj

```diff
@@ -0,0 +1,86 @@
+(ns ote.tasks.pre-notices-test
+  (:require [ote.tasks.pre-notices :as sut]
```

Comment:
":as pre-notices" would be easier to understand

Reply:
true, the "sut" name comes by default when creating test namespaces in CIDER and refers to System Under Test